.codecov.yml

```diff
@@ -1,2 +1,3 @@
 ignore:
     - "tests/*"
+    - "source/Internal/AssertHelper.cs"     # only used for Debug
```

Comment:
Path is not correct, as the [coverage](https://codecov.io/gh/gfoidl/Base64/commit/53843556c2f61fe6e038559511189716fe136b59) isn't increased, as expected.